numba_dpex/core/utils/kernel_launcher.py

```diff
@@ -68,15 +116,15 @@ def _build_array_attr_arg(
         ):
             array_attr = self.builder.load(array_attr)
 
-        self.build_arg(
+        self._build_arg(
             val=array_attr,
             ty=array_attr_ty,
             arg_list=arg_list,
             args_ty_list=args_ty_list,
             arg_num=arg_num,
         )
 
-    def _build_unituple_member_arg(
+    def _build_unituple_member_arg(  # pylint: disable=too-many-arguments
```

Comment:
Adding the pylint disabling is too much noise in code. Following linter guide line is OK, but there should be room to tweak.
For this particular check , can you update the `pylintrc` max-args to 6.

Reply:
I don't want to apply global settings for the number of max-args cause it affects readability. It is a todo work to update this code to use less arguments.